CHANGELOG.templates.md

```diff
@@ -26,11 +30,12 @@ Changes are grouped as follows:
 
 ### Added
 
-- Added Workflow with a Function and a Transformation to the cdf_data_pipeline_files_valhall example
+- Workflow with a Function and a Transformation to the cdf_data_pipeline_files_valhall example
   
 ### Changed
 
 - The `cdf_functions_dummy` module now includes codes from the former `common_function_code` directory.
+- Removed schedule from annotation function `context:files:oid:fileshare:annotation`

```

Comment:
Was this done earlier? Or was it done in this PR?

Reply:
this PR